### DISCUSSION TITLE
Rede Social Hashnode para desenvolvedores

### DISCUSSION THREAD

**Discussion by fabiomirandaa** [frontendbr/forum#368] [Dicas]
O que vocês acham da rede social Hashnode? É uma Ideia válida pra um network, ou pra não precisar ficar dependendo de grupos do Facebook? Pra quem não conhece o link é [esse](https://hashnode.com/)

**Comment by robsonbittencourt** [comment]
Não conhecia. Vou passar a utilizar.

**Comment by karlprieb** [comment]
Também não conhecia, vou dar uma olhada :)

**Comment by giioohbernini** [comment]
Nunca tinha visto falar, mas é uma ideia legal. Único problema é que já não estou dando conta de acompanhar as redes que eu participo, imagina mais uma? Hahaha, é muita informação saindo hoje em dia e em vários lugares diferentes. Acredito que toda nova plataforma/comunidade venha a somar para a nossa área, mas eu tenho um leve problema em gerenciar e filtrar isso :/

**Comment by vicainelli** [comment]
Interessante... no pouco que naveguei, me lembrou um pouco de Quora/Stackoverflow + Twitter + Facebook Groups
